lib/server.js

```diff
@@ -59,6 +60,9 @@ function serverFactory(core, messageHandler, Logger) {
             logger.info('proxyDHCP server is listening on %s:%s'.format(self.address, self.inPort));
             self.server.setBroadcast(true);
         });
+
+        var DHCPPoller = new IscDhcpLeasePoller({}, {});
+        DHCPPoller.run();
```

Comment:
Should this be a config option now that we have an ARP poller? This turns the DHCP poller back on by default.